### PR TITLE
feat: explore from here saved

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1732

### Description:
Explore from here on dashboards

This time I'm loadign a savedChart and don't allow overrides .

See this other PR for more details https://github.com/lightdash/lightdash/pull/1747#issuecomment-1096280877

### Preview:

Peek 2022-04-12 10-28

### Scope of the changes (select all that apply):
- [x] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

**Comment by rephus** [comment]
This is better than https://github.com/lightdash/lightdash/pull/1747#issuecomment-1096280877

**Comment by hamzahc1** [comment]
@rephus - in the PR preview these are the steps I followed: 

1. Open dashboard 
2. Add a new tile(saved chart) to dashboard 
3. Save dashbaord
4. Click on the icon on the saved chart tile -> Explore from here 
5. I then see this warning (even though I have saved the dashboard)
Screenshot 2022-04-12 at 12 20 31

**Comment by rephus** [comment]
> @rephus - in the PR preview these are the steps I followed:
> 
> 1. Open dashboard
> 2. Add a new tile(saved chart) to dashboard
> 3. Save dashbaord
> 4. Click on the icon on the saved chart tile -> Explore from here
> 5. I then see this warning (even though I have saved the dashboard)
> 
> Screenshot 2022-04-12 at 12 20 31

You were right @hamzahc1  there was a bug on the `window.addEventListener('beforeunload` (I was not removing the listener on changes) I made the fix on this PR, but let me know if you want the fix to be on a separate ticket/branch 
Peek 2022-04-12 15-22

**Comment by hamzahc1** [comment]
Change looks great and fixes it! QA complete 😄

File: packages/frontend/src/pages/Dashboard.tsx [hunk 1]
```diff
@@ -182,15 +182,18 @@ const Dashboard = () => {
...
+    const checkReload = (event: BeforeUnloadEvent) => {
         if (hasTilesChanged || haveFiltersChanged) {
             const message =
                 'You have unsaved changes to your dashboard! Are you sure you want to leave without saving?';
             event.returnValue = message;
             return message;
         }
-    });
+    };
+    useEffect(() => {
```
Comment (owlas) [inline packages/frontend/src/pages/Dashboard.tsx:193]: thanks for this fix

File: packages/frontend/src/components/Explorer.tsx [hunk 2]
```diff
@@ -107,6 +109,11 @@ export const Explorer: FC<Props> = ({ savedQueryUuid }) => {
         ChartType.CARTESIAN,
     );
 
+    const searchParams = new URLSearchParams(location.search);
+
+    const overrideQueryUuid: string | undefined = searchParams.get('explore')
```
Comment (owlas) [inline packages/frontend/src/components/Explorer.tsx:114]: such a great solution! Awesome @rephus

**Review by owlas — APPROVED** [review]

**Pull request merged**